src/main/java/racingcar/OutputView.java

```diff
@@ -0,0 +1,10 @@
+package racingcar;
+
+import java.util.List;
+
+public class OutputView {
+
+    public void displayResult(List listWinner) {
+        System.out.print("최종 우승자 : " + String.join(",", listWinner));
+    }
```

Comment:
요구사항은 항상 꼼꼼히!

> 요구 사항을 정확하게 준수한다
과제를 제출하기 전에 과제 진행 요구 사항, 기능 요구 사항, 프로그래밍 요구 사항을 모두 충족하였는지 다시 한번 확인한다. 이러한 요구 사항들은 미션마다 다르므로 항상 주의 깊게 읽어 본다.

```suggestion
    public void displayResult(List listWinner) {
        System.out.print("최종 우승자 : " + String.join(", ", listWinner));
    }
```

Reply:
그 Suggestion을 반영하시면 커밋이 남아요....
함부로 누르시면 안됩니다...